pinot-controller/src/main/java/org/apache/pinot/controller/BaseControllerStarter.java

```diff
@@ -501,6 +494,28 @@ protected void configure() {
 
     LOGGER.info("Starting controller admin application on: {}", ListenerConfigUtil.toString(_listenerConfigs));
     _adminApp.start(_listenerConfigs);
+    List<String> existingHlcTables = new ArrayList<>();
+    _helixResourceManager.getAllRealtimeTables().forEach(rt -> {
+      TableConfig tableConfig = _helixResourceManager.getTableConfig(rt);
+      if (tableConfig != null) {
+        Map<String, String> streamConfigMap = IngestionConfigUtils.getStreamConfigMap(tableConfig);
+        String streamType = streamConfigMap.getOrDefault(StreamConfigProperties.STREAM_TYPE, "kafka");
+        String consumerType = streamConfigMap.get(StreamConfigProperties.constructStreamProperty(streamType,
+            StreamConfigProperties.STREAM_CONSUMER_TYPES));
+        if (StreamConfig.ConsumerType.LOWLEVEL.name().equalsIgnoreCase(consumerType)
+            || "simple".equalsIgnoreCase(consumerType)) {
+          return;
+        } else {
+          existingHlcTables.add(rt);
+        }
+      }
+    });
+    if (existingHlcTables.size() > 0) {
+      LOGGER.error("High Level Consumer (HLC) based realtime tables are no longer supported. Please delete the "
+          + "following HLC tables before proceeding: \n");
+      existingHlcTables.forEach(s -> LOGGER.error("{}\n", s));
```

Comment:
Let's log it in one line
```suggestion
      LOGGER.error("High Level Consumer (HLC) based realtime tables are no longer supported. Please delete the "
          + "following HLC tables before proceeding: {}", existingHlcTables);
```